src/routes.jsx

```diff
@@ -6,8 +6,8 @@ import AndroidPage from './views/Android';
 import JsTeam from './views/JsTeam';
 import NimbledroidGraphPage from './views/NimbledroidGraph';
 import Quantum from './quantum/index';
```

Comment:
`import Quantum from './quantum';`

Reply:
https://deploy-preview-308--firefox-health-dashboard.netlify.com/quantum/tp6?bits=64&test=pageload  is expected to fail like this:  The URL has been fixed.

Concerning `import Quantum from './quantum/index';`  does your linter catch this?  Mine does not.

Reply:
I saw this when looking at the diff. Netlify previews ease the process of reviews such that I don't always need to pull the code down :)